net/lighttpd-https-cert/files/ssl-enable.conf

```diff
@@ -4,14 +4,22 @@
 # This package is not meant to be edited, it is part of package lighttpd-https-cert
 # If you need different https setup, uninstall this package and create your own configuration
 
+# As a special case in lighttpd 1.4.46 and later, if ssl.engine = "enable" in
+# the global scope and along with ssl.pemfile, then $SERVER["socket"] blocks
+# can inherit the exact global config by setting ssl.engine = "enable" and no
+# other ssl.* directives in the $SERVER["socket"] block.
+ssl.engine = "enable"
+ssl.pemfile = "/etc/lighttpd-self-signed.pem"
+ssl.use-sslv2 = "disable"
+ssl.use-sslv3 = "disable"
+ssl.cipher-list = "EDH+CAMELLIA:EDH+aECDSA:EECDH+aECDSA+AESGCM:EECDH+aECDSA+SHA384:EECDH+aECDSA+SHA256:EECDH:+CAMELLIA256:+AES256:+CAMELLIA128:+AES128:+SSLv3:!aNULL:!eNULL:!LOW:!3DES:!MD5:!EXP:!PSK:!DSS:!RC4:!SEED:CAMELLIA256-SHA:AES256-SHA:CAMELLIA128-SHA:AES128-SHA"
```

Comment:
This one is a bit tricky.

The default value is `HIGH`, however `HIGH` cipher list for OpenSSL library (1.0.2s) does not contain high enough ciphers from modern point of view. I think we should tune the list a bit but it would become deprecated soon and it will need an active maintenance of this package/configuration.

Extra focus must be given especially when upgrading from TOS 4 to TOS 5 and so. What do you think @Cynerd ? Can you propose some systematic way how to achieve this?

Cipher list itself is also a subject of discussion.

Reply:
These strong cipher suites are supported by OpenSSL 1.0.2t in both TOS 3.x and TOS 4.x
- ECDHE-ECDSA-AES128-GCM-SHA256
- ECDHE-RSA-AES128-GCM-SHA256
- ECDHE-ECDSA-AES256-GCM-SHA384
- ECDHE-RSA-AES256-GCM-SHA384
- DHE-RSA-AES128-GCM-SHA256
- DHE-RSA-AES256-GCM-SHA384

But we must ensure DHE parameters stronger than 1024 bits.

Reply:
Review of these cipher suites should be done along with OpenSSL upgrade (TOS 5.x?)